_includes/cockroachcloud/sso-intro.md

```diff
@@ -2,7 +2,7 @@ Single Sign-On (SSO) allows members of your {{ site.data.products.db }} organiza
 
 [Basic SSO](cloud-org-sso.html#basic-sso) is enabled by default for each {{ site.data.products.db }} organization. members can authenticate to [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) with any GitHub, Google, or Microsoft identity or with a password.
 
-[Cloud Organization SSO](cloud-org-sso.html#cloud-organization-sso) lets users sign in at a custom login page unique to your organization, and provides additional customization and capabilities to help your organization meet its security and compliance requirements. For example, it supports autoprovisioning, allows you to restrict the email addresses that can log in using a given method, and allows you to connect to your identity provider (IdP) using the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
+[Cloud Organization SSO](cloud-org-sso.html#cloud-organization-sso) lets users sign in at a custom login page unique to your organization, and provides additional customization and capabilities to help your organization meet its security and compliance requirements. For example, it supports autoprovisioning, allows you to restrict the email addresses that can log in using a given method, and allows you to connect to your identity provider (IdP) using the [Security Access Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language), [System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644), and [OpenID Connect (OIDC)](https://openid.net/connect/) identity protocols.
```

Comment:
SCIM is an identity management protocol whereas SAML and OIDC are auth protocols, so I would separate those from each other and not include SCIM in the list here